conductor/resources/resources.yml

```diff
@@ -16,6 +16,15 @@ instance_types:
     - {cores: 64, flavor: standard, description: "64 core, 240GB Mem"}
     - {cores: 64, flavor: highmem,  description: "64 core, 416GB Mem"}
 
+gpu_types:
+    - {gpu_count: 1,  gpu_flavor: k80,  description: "NVIDIA Tesla K80, 1 GPU"}
```

Comment:
Because `gpu_count` is `< 10`, you can remove one white space of indentation: `s/,  gpu_flavor/, gpu_flavor/`.